xla/service/gpu/cudnn_fused_mha_rewriter.cc

```diff
@@ -1113,19 +937,18 @@ MatchBwdResult MatchBwdBmmSoftmaxDropoutBmm(MatchBwdResult previous_result,
           kCudnnfMHASoftmaxBackwardCallTarget;
   }
   // try to pattern match dbias
-  HloInstruction* dS = has_mask ? d_mask : d_softmax;
+  HloInstruction* dS = d_softmax;
   if (dS->users()[0]->opcode() == HloOpcode::kConvert) {
     dS = dS->users()[0];
   }
   if (has_scale) {
-    // bmm1-(scale)-(bias)-(mask)-softmax pattern
-    // users could be dbias besides mask bwd or scale bwd
+    // bmm1-(scale)-(bias)-softmax pattern
```

Comment:
nit: I guess this comment can fit into one line? or at least more of the next line can fit in this line